examples/controls/src/app.js

```diff
@@ -33,8 +33,8 @@ export default class App extends Component {
         zoom: 3.5,
         bearing: 0,
         pitch: 0,
-        width: window.innerWidth,
-        height: window.innerHeight,
+        width: 500,
```

Comment:
Why does this have to default to 500 and not window.innerWidth? Is it because of server-side render?